llama_stack/distribution/server/server.py

```diff
@@ -484,5 +485,47 @@ def extract_path_params(route: str) -> List[str]:
     return params
 
 
+def is_flag_set(flag: str, args: List[str], default_args: argparse.Namespace) -> bool:
+    """
+    Checks if a specific flag is set in the provided arguments.
+
+    Args:
+        flag (str): The name of the flag to check.
+        args (List[str]): The list of arguments to check against.
+        default_args (argparse.Namespace): The default arguments to compare with.
+
+    Returns:
+        bool: True if the flag is set in the provided arguments, False otherwise.
+    """
+    return vars(default_args)[flag] != vars(args)[flag]
+
+
+def return_flag_value(
+    flag: str,
+    args: argparse.Namespace,
+    default_args: argparse.Namespace,
+    config_file_value: Optional[str],
+    env_var: Optional[str] = None,
+) -> str:
+    """
+    Determine the final value for a flag based on priority:
+    1. Environment Variable (ENV_VAR) → Highest priority
```

Comment:
(No action required) Guess I'm old fashioned but I prefer my code to not include non-ASCII codes.